### DISCUSSION TITLE
Link projects to github domain

### DISCUSSION THREAD

**Discussion by baeni** [community/community#21746] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Github Community!</p>
<p>I want to link i.e. my twitter profile to my github domain. That means I just want to make my twitter profile accessable with <a href="http://baeni.github.io/twitter" class="inline-onebox-loading" rel="nofollow noopener">baeni.github.io/twitter</a>. I am not that familar with domains and stuff so I hope someone can help me out with my problem.</p>
<p>Thanks!</p>
<p>Regards,</p>
<p>baeni</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>One option for this issue is to use a  <strong>META</strong> tag within your HTML page. Assuming you already have a page set for <em><a href="http://baeni.github.io/twitter" class="inline-onebox-loading" rel="nofollow noopener">baeni.github.io/twitter</a></em>, you could use:</p>
<pre><code>&lt;meta http-equiv="refresh" content="0; url=https:/twitter.com/yourtwitterusername"/&gt;
</code></pre>
<p>^ make sure that this  <strong>META </strong> tag goes within the  <strong>HEAD</strong> block</p>
<p>Also, for older browsers which don’t support this or if the refresh does not work properly, adding a redirect hyperlink will help combat this issue:</p>
<pre><code>&lt;p&gt;&lt;a href="https://twitter.com/yourtwitterusername"&gt;Redirect&lt;/a&gt;&lt;/p&gt;
</code></pre>
